package_readme.md

```diff
@@ -1,12 +1,10 @@
 <div align="center">
-  <img src="https://i.imgur.com/RUtiVzH.png" width="600" /><br><br>
+  <img src="https://i.imgur.com/dQLeGCc.png" width="600" /><br><br>
```

Comment:
Ideally, we should get rid of a separate file for PyPI, just didn't want to think about that in this PR.